package-lock.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Shouldn't this have more stuff? I don't understand the need of the lock file tbh.